packages/styles/breadcrumb.css

```diff
@@ -21,13 +21,17 @@
 }
 
 .Breadcrumb__Link {
-  font-weight: 400;
+  font-weight: inherit;
   padding: 0;
   display: inline;
 }
 
 .Breadcrumb__Item {
-  font-weight: 500;
+  font-weight: var(--font-weight-medium);
   color: var(--link-text-color);
   display: inline;
 }
+
+.Breadcrumb__Item:not(:last-child) {
+  font-weight: var(--font-weight-semi-bold);
```

Comment:
2 things.

This is the wrong font-weight and reads a little bit weird with the above rule. The "standard" font-weight for all of the breadcrumb items should be `--font-weight-medium` which is correct on line 30 above.

The _last_ item in the breadcrumb has the normal font-weight, this can be rewritten as:

```css
.Breadcrumb__Item:last-child {
  font-weight: var(--font-weight-normal);
}
```